### ISSUE TITLE
Manual Overhaul

### ISSUE THREAD

**Issue by porres** [pure-data/pddp#225]
I think Pd's manual is not well structured.

It starts with the inner workings of Pd, then it goes into installing and configuring, but configuration is still not quite complete. Then, a last chapter is about building it.

It makes sense to me that a manual starts with installation. It is also intuitive to me to add a how to build pd section after the download and install, as these are related. So we'd start with this chapter on installing/building, and then go to a chapter about opening and configuring it. And this configuration chapter could include all tabs and things that we still miss.

Then, finally, we can have the so called "theory of operation" as a final stage.

I see that installing externals can be an appendix that comes after that, followed by "current state/release notes".
So, this is what it would look like

1- Introduction
2- Installing and building Pure Data
3- Configuring and running Pd
4- Theory of operation
5- Appendix I: Install Externals
6- Appendix II: Release Notes

**Comment by millerpuckette** [comment]
I think this all makes sense - although perhaps how to build Pd should 
also be an appendix - most readers won't want to confront that at all.

And perhaps "theory of operation" should eventually be two sections, 
"using Pd" including editing patches, managing files, and an intro to 
the various kinds of objects; and another, real theory of operations 
that explains stuff like message passing, "$" expansions, DSP chain 
sorting, order of execution of messages, and all that...

cheers

Miller

**Comment by porres** [comment]
yeah, building Pd is already detached and into a section we can also describe as an 'appendix'. So here's another skeleton sketch

Chapter 1: - Introduction
Chapter 2: - Installing and starting Pd
Chapter 3: - Configuring Pd
Chapter 4: - Basic operation (opening, running and editing patches)
Chapter 5: - Advanced operation (Syntax and Theory of operation)
Appendix I: - Installing Externals
Appendix II: - Building Pd from source
Appendix III: - Release Notes

We can do this with a simple reordering of the elements without adding or subtracting anything.

**Comment by millerpuckette** [comment]
I would suggest distinguishing between "how to use Pd" and "how Pd 
works" (theory of operation) - the one is what you do with it and the 
other is what it does.  I think that would be clearer than mixing the 
two in together (as I originally did somewhat :)

But neither is more "advanced" than the other - they're just different 
matters.

However, there might be other, better ways to subdivide things - I'm not 
the greatest technical writer...

cheers

Miller

**Comment by porres** [comment]
Maybe @ben-wes also has something to say

anyway, I see what you mean and looking at what we have, chapter 4 would be basically an overview and then sections 2.2 and 2.3 on editing Pd patches. I also felt like this could be detached from the rest of what we have there, to be honest. Therefore, Chapter 5 would be the remaining of Chapter 2.

After the reorganization of all contents into this new format, we can think about other things to add and stuff

so, the new plan:

Chapter 1: - Introduction
Chapter 2: - Installing and starting Pd
Chapter 3: - Configuring Pd
Chapter 4: - Running and Creating/Editing Pd patches
Chapter 5: - Theory of operation (Syntax and how Pd Works)
Appendix I: - Installing Externals
Appendix II: - Building Pd from source
Appendix III: - Release Notes

**Comment by millerpuckette** [comment]
Looks great to me!

cheers

M
